jupyter_dash/jupyter_app.py

```diff
@@ -356,18 +378,12 @@ def _config_callback_exception_handling(
     ):
 
         @self.server.errorhandler(Exception)
-        def _wrap_errors(_):
+        def _wrap_errors(error):
             """Install traceback handling for callbacks"""
             self._traceback = sys.exc_info()[2]
 
             # Compute number of stack frames to skip to get down to callback
```

Comment:
You have the traceback here or in `error.__traceback__`, can extract the text with the `traceback` module.

Reply:
Got it!